src/remote.go

```diff
@@ -95,7 +95,7 @@ func genVulns(mc *metaConfig, id *imageData) string {
 
 	// Build vuln string
 	for _, penalty := range id.Penalties {
-		vulnString.WriteString(fmt.Sprintf("[PENALTY] %s - %.0f pts", penalty.Message, math.Abs(float64(penalty.Points))))
+		vulnString.WriteString(fmt.Sprintf("[PENALTY] %s - N%.0f pts", penalty.Message, math.Abs(float64(penalty.Points))))
```

Comment:
So, in remote I was splitting on `-` and taking the last section of the split (since we have complete control over appending `- X pts`). However, negative values make that difficult, so instead of `-`, I added an `N` to indicate that it was negative. You can see the handling in remote [here](https://github.com/sourque/sarpedon/commit/ec9edca26009131a009c5ed03073002e1dcbb70e).

Reply:
Got it